RecoParticleFlow/Configuration/python/RecoParticleFlow_EventContent_cff.py

```diff
@@ -110,18 +19,23 @@
     'keep recoPFRecHits_particleFlowRecHitHF_Cleaned_*',
     'keep recoPFRecHits_particleFlowRecHitHO_Cleaned_*',
     'keep recoPFRecHits_particleFlowRecHitPS_Cleaned_*',
-    #'keep recoPFCandidates_*_*_*',
     'keep recoCaloClusters_particleFlowEGamma_*_*',
     'keep recoSuperClusters_particleFlowEGamma_*_*',
     'keep recoCaloClusters_particleFlowSuperClusterECAL_*_*',
     'keep recoSuperClusters_particleFlowSuperClusterECAL_*_*',
     'keep recoConversions_particleFlowEGamma_*_*',
     'keep recoPFCandidates_particleFlow_*_*',
-    'keep recoPFCandidates_particleFlowTmp_*_*',
-    'drop recoPFCandidates_particleFlowTmp__*',
+    'keep recoPFCandidates_particleFlowTmp_AddedMuonsAndHadrons_*',
+    'keep recoPFCandidates_particleFlowTmp_CleanedCosmicsMuons_*',
+    'keep recoPFCandidates_particleFlowTmp_CleanedFakeMuons_*',
+    'keep recoPFCandidates_particleFlowTmp_CleanedHF_*',
+    'keep recoPFCandidates_particleFlowTmp_CleanedPunchThroughMuons_*',
+    'keep recoPFCandidates_particleFlowTmp_CleanedPunchThroughNeutralHadrons_*',
+    'keep recoPFCandidates_particleFlowTmp_CleanedTrackerAndGlobalMuons_*',
     'keep *_particleFlow_electrons_*',
     'keep *_particleFlow_photons_*',
     'keep *_particleFlow_muons_*',
+    'drop *_pfElectronTranslator_*_*', #please check pf*Translator really use in AOD
```

Comment:
This is probably not needed, as all the corresponding ```keep``` statements are only after it